ember-cli-build.js

```diff
@@ -23,11 +23,6 @@ module.exports = function (defaults) {
     sassOptions: {
       includePaths: ['node_modules/@appuniversum/ember-appuniversum'],
     },
-    babel: {
-      plugins: [
-        require.resolve('ember-concurrency/async-arrow-task-transform'),
```

Comment:
Why did we have to remove this? Is this that was added to babbel by default or so?